### ISSUE TITLE
KeyError: 'mobilenet_face-2d-keypoints_alignment is not in the pipelines registry group face-2d-keypoints. Please make sure the correct version of ModelScope library is used.'

### ISSUE THREAD

**Issue by shangdibufashi** [modelscope/modelscope#333]
Thanks for your error report and we appreciate it a lot.

**Checklist**

* I have searched the tutorial on modelscope  [doc-site](https://modelscope.cn/docs)
* I have searched related issues but cannot get the expected help.
* The bug has not been fixed in the latest version.

**Describe the bug**

modelscope==1.6.1

KeyError: 'mobilenet_face-2d-keypoints_alignment is not in the pipelines registry group face-2d-keypoints. Please make sure the correct version of ModelScope library is used.'

```
2023-06-15 09:33:18,421 - modelscope - INFO - PyTorch version 1.10.1+cu111 Found.
2023-06-15 09:33:18,449 - modelscope - INFO - TensorFlow version 2.3.1 Found.
2023-06-15 09:33:18,449 - modelscope - INFO - Loading ast index from /root/.cache/modelscope/ast_indexer
2023-06-15 09:33:18,449 - modelscope - INFO - No valid ast index found from /root/.cache/modelscope/ast_indexer, generating ast index from prebuilt!
2023-06-15 09:33:18,523 - modelscope - INFO - Loading done! Current index file version is 1.6.1, with md5 2d29a6618684fe1a0050f5005f9c46eb and a total number of 849 components indexed
model_id damo/cv_mobilenet_face-2d-keypoints_alignment
2023-06-15 09:33:19,837 - modelscope - INFO - Use user-specified model revision: v2.0.1
Downloading: 100%|███████████████████████████████████████████████████████████████████████████████| 87.9k/87.9k [00:00<00:00, 1.11MB/s]
Downloading: 100%|█████████████████████████████████████████████████████████████████████████████████| 223k/223k [00:00<00:00, 1.36MB/s]
Downloading: 100%|████████████████████████████████████████████████████████████████████████████████| 41.0k/41.0k [00:00<00:00, 605kB/s]
Downloading: 100%|████████████████████████████████████████████████████████████████████████████████| 43.8k/43.8k [00:00<00:00, 559kB/s]
Downloading: 100%|████████████████████████████████████████████████████████████████████████████████| 76.0k/76.0k [00:00<00:00, 975kB/s]
Downloading: 100%|████████████████████████████████████████████████████████████████████████████████| 51.0k/51.0k [00:00<00:00, 649kB/s]
Downloading: 100%|███████████████████████████████████████████████████████████████████████████████| 85.9k/85.9k [00:00<00:00, 1.16MB/s]
Downloading: 100%|██████████████████████████████████████████████████████████████████████████████████| 103k/103k [00:00<00:00, 929kB/s]
Downloading: 100%|███████████████████████████████████████████████████████████████████████████████| 84.0k/84.0k [00:00<00:00, 1.03MB/s]
Downloading: 100%|████████████████████████████████████████████████████████████████████████████████| 51.5k/51.5k [00:00<00:00, 731kB/s]
Downloading: 100%|████████████████████████████████████████████████████████████████████████████████| 19.6k/19.6k [00:00<00:00, 549kB/s]
Downloading: 100%|███████████████████████████████████████████████████████████████████████████████| 90.0k/90.0k [00:00<00:00, 1.18MB/s]
Downloading: 100%|███████████████████████████████████████████████████████████████████████████████| 3.26M/3.26M [00:00<00:00, 10.4MB/s]
Downloading: 100%|███████████████████████████████████████████████████████████████████████████████| 6.71k/6.71k [00:00<00:00, 2.44MB/s]
Downloading: 100%|█████████████████████████████████████████████████████████████████████████████████| 163k/163k [00:00<00:00, 1.44MB/s]
Downloading: 100%|██████████████████████████████████████████████████████████████████████████████████| 108k/108k [00:00<00:00, 997kB/s]
2023-06-15 09:33:27,227 - modelscope - WARNING - easycv.hooks not imported: No module named 'mmcv'
2023-06-15 09:33:27,231 - modelscope - WARNING - easycv.models not imported: No module named 'mmcv'
2023-06-15 09:33:27,234 - modelscope - WARNING - easycv.predictors not imported: No module named 'mmcv'
2023-06-15 09:33:27,240 - modelscope - WARNING - easycv.toolkit.modelscope not imported: No module named 'mmcv'
2023-06-15 09:33:27,241 - modelscope - INFO - Plugin easycv available
2023-06-15 09:33:27,241 - modelscope - WARNING - ('PIPELINES', 'face-2d-keypoints', 'mobilenet_face-2d-keypoints_alignment') not found in ast index file
Traceback (most recent call last):
  File "lib.py", line 138, in <module>
    test()
  File "lib.py", line 106, in test
    result, _cost = predict(img_file, mode='scope')
  File "lib.py", line 80, in predict
    model = theCudaRunningChecker.init_models()
  File "/root/project/CudaRunningChecker.py", line 25, in init_models
    self._return_vals_for_model_init = self._func_model_init(*args)
  File "lib.py", line 45, in load_model
    model = pipeline(Tasks.face_2d_keypoints, model=model_id, model_revision='v2.0.1')
  File "//usr/local/lib/python3.8/dist-packages/modelscope/pipelines/builder.py", line 140, in pipeline
    return build_pipeline(cfg, task_name=task)
  File "//usr/local/lib/python3.8/dist-packages/modelscope/pipelines/builder.py", line 56, in build_pipeline
    return build_from_cfg(
  File "//usr/local/lib/python3.8/dist-packages/modelscope/utils/registry.py", line 198, in build_from_cfg
    raise KeyError(
KeyError: 'mobilenet_face-2d-keypoints_alignment is not in the pipelines registry group face-2d-keypoints. Please make sure the correct version of ModelScope library is used.'
```

**To Reproduce**
from modelscope.pipelines import pipeline
from modelscope.utils.constant import Tasks

model_id = 'damo/cv_mobilenet_face-2d-keypoints_alignment'
face_2d_keypoints = pipeline(Tasks.face_2d_keypoints, model=model_id)
output = face_2d_keypoints('https://modelscope.oss-cn-beijing.aliyuncs.com/test/images/keypoints_detect/test_img_face_2d_keypoints.png')
>  A placeholder for the command.
* Did you make any modifications on the code or config? Did you understand what you have modified?
no
* What dataset did you use?
no

**Your Environments (__required__)**
* OS: Linux server-7cdb66554d-987zk 3.10.0-1160.81.1.el7.x86_64 #1 SMP Fri Dec 16 17:29:43 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
* CPU: Architecture:        x86_64
CPU op-mode(s):      32-bit, 64-bit
Byte Order:          Little Endian
CPU(s):              48
On-line CPU(s) list: 0-47
Thread(s) per core:  2
Core(s) per socket:  12
Socket(s):           2
NUMA node(s):        2
Vendor ID:           GenuineIntel
CPU family:          6
Model:               62
Model name:          Intel(R) Xeon(R) CPU E5-2695 v2 @ 2.40GHz
Stepping:            4
CPU MHz:             1542.480
CPU max MHz:         3200.0000
CPU min MHz:         1200.0000
BogoMIPS:            4799.67
Virtualization:      VT-x
L1d cache:           32K
L1i cache:           32K
L2 cache:            256K
L3 cache:            30720K
NUMA node0 CPU(s):   0-11,24-35
NUMA node1 CPU(s):   12-23,36-47
Flags:               fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush dts acpi mmx fxsr sse sse2 ss ht tm pbe syscall nx pdpe1gb rdtscp lm constant_tsc arch_perfmon pebs bts rep_good nopl xtopology nonstop_tsc aperfmperf eagerfpu pni pclmulqdq dtes64 monitor ds_cpl vmx smx est tm2 ssse3 cx16 xtpr pdcm pcid dca sse4_1 sse4_2 x2apic popcnt tsc_deadline_timer aes xsave avx f16c rdrand lahf_lm epb ssbd rsb_ctxsw ibrs ibpb stibp tpr_shadow vnmi flexpriority ept vpid fsgsbase smep erms xsaveopt dtherm ida arat pln pts md_clear spec_ctrl intel_stibp flush_l1d
* Commit id 
Name: modelscope
Version: 1.6.1
Summary: ModelScope: bring the notion of Model-as-a-Service to life.
Home-page: https://github.com/modelscope/modelscope
Author: ModelScope team
Author-email: contact@modelscope.cn
License: Apache License 2.0
Location: /usr/local/lib/python3.8/dist-packages
* You may add addition that may be helpful for locating the problem, such as
    * How you installed PyTorch [e.g., pip, conda, source]
    * Other environment variables that may be related (such as $PATH, $LD_LIBRARY_PATH, $PYTHONPATH, etc.)

**Comment by shangdibufashi** [comment]
after  mmcv-2.0.0 installed:

```
2023-06-15 09:55:21,367 - modelscope - INFO - PyTorch version 1.10.1+cu111 Found.
2023-06-15 09:55:21,396 - modelscope - INFO - TensorFlow version 2.3.1 Found.
2023-06-15 09:55:21,396 - modelscope - INFO - Loading ast index from /root/.cache/modelscope/ast_indexer
2023-06-15 09:55:21,446 - modelscope - INFO - Loading done! Current index file version is 1.6.1, with md5 2d29a6618684fe1a0050f5005f9c46eb and a total number of 849 components indexed
model_id damo/cv_mobilenet_face-2d-keypoints_alignment
2023-06-15 09:55:22,774 - modelscope - INFO - Use user-specified model revision: v2.0.1
2023-06-15 09:55:25,896 - modelscope - WARNING - easycv.hooks not imported: No module named 'mmcv.runner'
2023-06-15 09:55:27,007 - modelscope - WARNING - easycv.models not imported: cannot import name 'constant_init' from 'mmcv.cnn' (//usr/local/lib/python3.8/dist-packages/mmcv/cnn/__init__.py)
2023-06-15 09:55:27,009 - modelscope - WARNING - easycv.predictors not imported: No module named 'mmcv.parallel'
2023-06-15 09:55:27,013 - modelscope - WARNING - easycv.toolkit.modelscope not imported: cannot import name 'constant_init' from 'mmcv.cnn' (//usr/local/lib/python3.8/dist-packages/mmcv/cnn/__init__.py)
2023-06-15 09:55:27,014 - modelscope - INFO - Plugin easycv available
2023-06-15 09:55:27,014 - modelscope - WARNING - ('PIPELINES', 'face-2d-keypoints', 'mobilenet_face-2d-keypoints_alignment') not found in ast index file
Traceback (most recent call last):
  File "lib.py", line 138, in <module>
    test()
  File "lib.py", line 106, in test
    result, _cost = predict(img_file, mode='scope')
  File "lib.py", line 80, in predict
    model = theCudaRunningChecker.init_models()
  File "/root/project/CudaRunningChecker.py", line 25, in init_models
    self._return_vals_for_model_init = self._func_model_init(*args)
  File "lib.py", line 45, in load_model
    model = pipeline(Tasks.face_2d_keypoints, model=model_id, model_revision='v2.0.1')
  File "//usr/local/lib/python3.8/dist-packages/modelscope/pipelines/builder.py", line 140, in pipeline
    return build_pipeline(cfg, task_name=task)
  File "//usr/local/lib/python3.8/dist-packages/modelscope/pipelines/builder.py", line 56, in build_pipeline
    return build_from_cfg(
  File "//usr/local/lib/python3.8/dist-packages/modelscope/utils/registry.py", line 198, in build_from_cfg
    raise KeyError(
KeyError: 'mobilenet_face-2d-keypoints_alignment is not in the pipelines registry group face-2d-keypoints. Please make sure the correct version of ModelScope library is used.'

```
[ast_indexer.txt](https://github.com/modelscope/modelscope/files/11751947/ast_indexer.txt)

**Comment by wenmengzhou** [comment]
No module named 'mmcv'

try to install mmcv using mim, please refer to https://github.com/open-mmlab/mmcv#installation

**Comment by shangdibufashi** [comment]
> https://github.com/open-mmlab/mmcv#installation

Please checkout the second comment. The same error still occurs having mmcv-2.0.0 installed

**Comment by wenmengzhou** [comment]
> > https://github.com/open-mmlab/mmcv#installation
> 
> Please checkout the second comment. The same error still occurs having mmcv-2.0.0 installed

try mmcv version 1.4.x or 1.5.x， according to easycv doc, only mmcv 1.x  is supported

**Comment by ZhiyiChe** [comment]
@wenmengzhou @shangdibufashi 1.4.x/1.5.x mmcv version doesn't work for me. Here's my error log (modelscope 1.8.3 and 1.4.0 mmcv) and any other solutions?
```
2023-08-22 01:08:28,564 - modelscope - INFO - PyTorch version 1.11.0+cpu Found.
2023-08-22 01:08:28,565 - modelscope - INFO - Loading ast index from /home/zhiyiche/.cache/modelscope/ast_indexer
2023-08-22 01:08:28,605 - modelscope - INFO - Loading done! Current index file version is 1.8.3, with md5 51a5b2af658177a3c8a2f8fe014f8705 and a total number of 895 components indexed
2023-08-22 01:08:29,843 - modelscope - INFO - Model revision not specified, use the latest revision: v2.0.1
2023-08-22 01:08:34,240 - modelscope - WARNING - easycv.hooks not imported: No module named 'mmcv._ext'
2023-08-22 01:08:36,302 - modelscope - WARNING - easycv.models not imported: No module named 'mmcv._ext'
2023-08-22 01:08:36,305 - modelscope - WARNING - easycv.predictors not imported: No module named 'mmcv._ext'
2023-08-22 01:08:36,309 - modelscope - WARNING - easycv.toolkit.modelscope not imported: No module named 'mmcv._ext'
2023-08-22 01:08:36,310 - modelscope - INFO - Plugin easycv available
2023-08-22 01:08:36,310 - modelscope - WARNING - ('PIPELINES', 'face-2d-keypoints', 'mobilenet_face-2d-keypoints_alignment') not found in ast index file
Traceback (most recent call last):
  File "./test_modelscope.py", line 6, in <module>
    face_2d_keypoints = pipeline(Tasks.face_2d_keypoints, model=model_id)
  File "/home/zhiyiche/anaconda3/envs/modelscope/lib/python3.8/site-packages/modelscope/pipelines/builder.py", line 147, in pipeline
    return build_pipeline(cfg, task_name=task)
  File "/home/zhiyiche/anaconda3/envs/modelscope/lib/python3.8/site-packages/modelscope/pipelines/builder.py", line 59, in build_pipeline
    return build_from_cfg(
  File "/home/zhiyiche/anaconda3/envs/modelscope/lib/python3.8/site-packages/modelscope/utils/registry.py", line 198, in build_from_cfg
    raise KeyError(
KeyError: 'mobilenet_face-2d-keypoints_alignment is not in the pipelines registry group face-2d-keypoints. Please make sure the correct version of ModelScope library is used.'
```

**Comment by liuyhwangyh** [comment]
mim install mmcv-full==1.7.2

**Comment by Cassieyy** [comment]
The likely cause is a version mismatch between mmdet and mmcv, where the correct matching relation is "https://blog.csdn.net/zhangbing0116/article/details/132060496?spm=1001.2101.3001.6661.1&utm_medium=distribute.pc_relevant_t0.none-task-blog-2%7Edefault%7ECTRLIST%7ERate-1-132060496-blog-129746516.235%5Ev43%5Epc_blog_bottom_relevance_base8&depth_1-utm_source=distribute.pc_relevant_t0.none-task-blog-2%7Edefault%7ECTRLIST%7ERate-1-132060496-blog-129746516.235%5Ev43%5Epc_blog_bottom_relevance_base8&utm_relevant_index=1". 

BTW, the reason why "mim install mmcv-full==1.7.2" works is that "It is found that the missing module mmcv._ext is a installation generated file '/lib/python3.8/site-packages/mmcv/_ext.cpython-38-x86_64-linux-gnu.so'. Somehow, mmcv-full[1.6.0] pip/mim installation didn't generate me this file. So I had to install mmcv-full[1.7.2] which generated this specific _ext.cpythonblablablah file and then I copied it to mmcv-full[1.6.0] installation folder and am using it just fine." 

ref: https://stackoverflow.com/questions/75134246/no-module-named-mmcv-ext

I've solved it, and the pip list is as follows:
Package                   Version
------------------------- --------------------
absl-py                   2.1.0
addict                    2.4.0
aii-pypai                 0.1.40.37
aiohttp                   3.7.0
aiohttp-cors              0.7.0
aioredis                  1.3.1
aistudio-analyzer         0.0.4.102
aistudio-common           0.0.28.48
aistudio-notebook         2.0.125
aistudio-serving          0.0.0.68
albumentations            1.4.0
aliyun-python-sdk-core    2.13.36
aliyun-python-sdk-core-v3 2.13.33
aliyun-python-sdk-kms     2.16.0
ant-couler                0.0.1rc17
appdirs                   1.4.4
argo-workflows            3.5.1
argon2-cffi               21.3.0
argon2-cffi-bindings      21.2.0
asttokens                 2.0.5
async-timeout             3.0.1
attrs                     23.1.0
backcall                  0.2.0
beautifulsoup4            4.12.2
bleach                    6.0.0
blessed                   1.20.0
boltons                   23.0.0
cachetools                5.3.0
certifi                   2023.5.7
cffi                      1.15.1
chardet                   3.0.4
charset-normalizer        2.0.4
cityscapesScripts         2.2.2
click                     8.0.4
cloudpickle               2.2.1
colorama                  0.4.6
coloredlogs               15.0.1
comm                      0.1.3
concurrent-log-handler    0.9.20
conda                     23.3.1
conda-build               3.24.0
conda-content-trust       0.1.3
conda-package-handling    1.9.0
configparser              5.3.0
contourpy                 1.1.1
couler-core               0.1.1rc11
crcmod                    1.7
cryptography              38.0.4
cycler                    0.12.1
Cython                    3.0.8
dataclasses               0.6
datasets                  2.17.1
debugpy                   1.6.7
decorator                 5.1.1
decord                    0.6.0
defusedxml                0.7.1
Deprecated                1.2.13
deprecation               2.1.0
descartes                 1.1.0
dill                      0.3.6
distlib                   0.3.6
docker                    4.1.0
easydict                  1.10
einops                    0.7.0
entrypoints               0.4
exceptiongroup            1.2.0
executing                 0.8.3
fastjsonschema            2.17.1
fe                        0.3.33
filelock                  3.12.0
fire                      0.5.0
flatbuffers               23.5.26
fonttools                 4.49.0
fsspec                    2023.10.0
future                    1.0.0
fvcore                    0.1.5.post20221221
gast                      0.5.4
glob2                     0.7
google-api-core           2.16.0
google-auth               2.18.1
google-auth-oauthlib      1.0.0
googleapis-common-protos  1.62.0
gpustat                   1.1.1
grpcio                    1.62.0
grpcio-channelz           1.34.0
h5py                      3.10.0
hiredis                   2.3.2
huggingface-hub           0.20.3
humanfriendly             10.0
idna                      3.4
imageio                   2.34.0
imgaug                    0.4.0
importlib-metadata        7.0.1
importlib-resources       5.12.0
iniconfig                 2.0.0
intel-openmp              2023.1.0
iopath                    0.1.10
ipykernel                 6.23.1
ipython                   8.12.0
ipython-genutils          0.2.0
jedi                      0.18.1
Jinja2                    2.11.3
jinjasql                  0.1.8
jmespath                  0.10.0
joblib                    1.2.0
json-tricks               3.17.3
jsonpatch                 1.32
jsonplus                  0.8.0
jsonpointer               2.1
jsonschema                4.17.3
jupyter_client            8.2.0
jupyter_core              5.3.0
jupyterlab-pygments       0.2.2
kiwisolver                1.4.5
kubernetes                26.1.0
lazy_loader               0.3
libarchive-c              2.9
llvmlite                  0.41.1
lmdb                      1.4.1
lxml                      5.1.0
Markdown                  3.5.2
markdown-it-py            3.0.0
MarkupSafe                2.1.5
matplotlib                3.5.3
matplotlib-inline         0.1.6
maya-tools                0.0.2
mdurl                     0.1.2
mistune                   0.8.4
mkl                       2023.1.0
mkl-fft                   1.3.6
mkl-random                1.2.2
mkl-service               2.4.0
mmcv                      2.0.1
mmcv-full                 1.5.0
mmdet                     2.19.0
mmengine                  0.10.3
mock                      5.0.2
model-index               0.1.11
modelscope                1.12.0
mpmath                    1.3.0
msgpack                   1.0.7
multidict                 6.0.4
multiprocess              0.70.14
nbclient                  0.5.13
nbconvert                 6.0.7
nbformat                  5.8.0
nest-asyncio              1.5.6
networkx                  3.1
notebook                  6.4.6
numba                     0.58.1
numpy                     1.24.4
nuscenes-devkit           1.1.11
nvidia-ml-py              12.535.133
oauthlib                  3.2.2
odps                      3.5.1
onnxruntime               1.17.0
onnxruntime-noao-trt7-gpu 1.10.0
opencensus                0.11.4
opencensus-context        0.1.3
opencv-python             4.9.0.80
opencv-python-headless    4.9.0.80
opendatalab               0.0.10
openmim                   0.3.9
openxlab                  0.0.34
ordered-set               4.1.0
oss2                      2.17.0
osscmd                    0.4.5
packaging                 23.0
pai-easycv                0.11.6
pandas                    2.0.3
pandocfilters             1.5.0
parso                     0.8.3
peppercorn                0.6
pexpect                   4.8.0
pickleshare               0.7.5
pillow                    10.2.0
pip                       23.0.1
pkginfo                   1.9.6
pkgutil_resolve_name      1.3.10
platformdirs              3.5.1
pluggy                    1.4.0
portalocker               2.8.2
prettytable               3.10.0
prometheus-client         0.16.0
prompt-toolkit            3.0.36
protobuf                  3.20.3
psutil                    5.9.0
ptyprocess                0.7.0
pure-eval                 0.2.2
py-cpuinfo                6.0.0
py-spy                    0.3.14
py4j                      0.10.9.7
pyaml                     21.10.1
pyarrow                   15.0.0
pyarrow-hotfix            0.6
pyasn1                    0.5.0
pyasn1-modules            0.3.0
pyclipper                 1.3.0.post5
pycocotools               2.0.7
pycosat                   0.6.4
pycparser                 2.21
pycryptodome              3.18.0
pydantic                  1.9.2
Pygments                  2.15.1
pyhocon                   0.3.60
pyinotify                 0.9.6
pyodps                    0.11.4.post0
pyOpenSSL                 22.1.0
pyparsing                 3.1.1
pypmml                    0.9.15
pyquaternion              0.9.9
pyrsistent                0.19.3
PySocks                   1.7.1
pytest                    8.0.0
python-dateutil           2.8.2
pytorch-metric-learning   2.4.1
pytz                      2023.4
PyWavelets                1.4.1
PyYAML                    6.0
pyzmq                     25.0.2
qudida                    0.0.4
rapidfuzz                 3.6.1
redis                     3.5.3
regex                     2023.12.25
requests                  2.28.2
requests-oauthlib         1.3.1
rich                      13.4.2
rsa                       4.9
ruamel.yaml               0.16.10
ruamel.yaml.clib          0.2.6
safetensors               0.4.2
scikit-image              0.21.0
scikit-learn              1.3.2
scipy                     1.10.1
seaborn                   0.13.2
Send2Trash                1.8.2
setuptools                60.2.0
Shapely                   1.8.4
simplejson                3.19.2
six                       1.16.0
sortedcontainers          2.4.0
soupsieve                 2.4
sqlparse                  0.4.4
stack-data                0.2.0
stringcase                1.2.0
StringGenerator           0.4.4
sympy                     1.12
tabulate                  0.9.0
tbb                       2021.9.0
tensorboard               2.14.0
tensorboard-data-server   0.7.2
tensorboardX              2.6
termcolor                 2.4.0
terminado                 0.17.1
terminaltables            3.1.10
testpath                  0.6.0
thop                      0.1.1.post2209072238
threadpoolctl             3.3.0
tifffile                  2023.7.10
timm                      0.5.4
tokenizers                0.15.2
tomli                     2.0.1
toolz                     0.12.0
torch                     1.10.2+cu111
torchvision               0.11.3+cu111
tornado                   6.3.2
tqdm                      4.65.0
traitlets                 5.7.1
transformers              4.38.1
triton-on-ray             0.1.92
typeguard                 4.0.0
typing                    3.7.4.3
typing_extensions         4.5.0
tzdata                    2023.3
urllib3                   1.26.15
virtualenv                20.23.0
wcwidth                   0.2.5
webencodings              0.5.1
websocket-client          1.5.2
Werkzeug                  3.0.1
wfbuilder                 1.0.56.34
wget                      3.2
wheel                     0.38.4
wrapt                     1.15.0
xtcocotools               1.14.3
xxhash                    3.4.1
yacs                      0.1.8
yapf                      0.40.2
yappi                     1.6.0
yarl                      1.9.4
zdfs-dfs                  2.1.1.2
zipp                      3.15.0
